packages/shared/src/utils/validation/form-validations.ts

```diff
@@ -111,4 +111,4 @@ export const getValidationFunction = (rule: string) => {
 
 // Adding string as type because, employment_status can come from Personal details or Financial assessment.
 export const shouldHideOccupationField = (employment_status?: TEmploymentStatus | string) =>
```

Comment:
should we rename this to a better variable name? 🤔

Reply:
It's being used multiple places, you can ignore my comment and remain the same variable name 🙏